### ISSUE TITLE
labels in IMBD

### ISSUE THREAD

**Issue by AlexanderTrg** [pytorch/text#2162]
## 🐛 Bug
all labels in train ds from IMBD are 1

from torchtext.datasets import IMDB
train_iter = IMDB(split='train') 
labels = [label for label, text in train_iter]
print(sum(labels)/len(labels))
1.0

pytorch 2.0.0+cu118 
torchtext 0.15.1+cpu

**Comment by Nayef211** [comment]
Closing because this is a duplicate of https://github.com/pytorch/text/issues/2041. Let's track the issue there
